docs/guides/continuous-integration/circleci.mdx

```diff
@@ -11,14 +11,16 @@ title: CircleCI
 
 :::
 
-<!-- textlint-disable -->
-
-<DocsVideo src="https://youtube.com/embed/J-xbNtKgXfY" />
-
-<DocsVideo src="/img/snippets/running-in-ci.mp4" title="Cypress in CircleCI" />
-
 <!-- textlint-enable -->
 
+The
+[Cypress CircleCI Orb](https://circleci.com/developer/orbs/orb/cypress-io/cypress)
+is the _official_ CircleCI Orb of [Cypress.io](https://cypress.io). Although you
+do not need to use the orb to run your tests in CircleCI, the benefit of using
+the orb is that it allows you to easily install, cache and run Cypress tests in
+CircleCI with less effort. The orb abstracts common steps necessary for running
+your tests in CircleCI without an orb to make your life as a developer better!
+
 ## Basic Setup
 
 The [Cypress CircleCI Orb](https://github.com/cypress-io/circleci-orb) is a
```

Comment:
I think we pull the line "Full documentation can be found at the
[`cypress-io/circleci-orb`](https://github.com/cypress-io/circleci-orb) repo." since we're going to link the new readme to the Circle registry listing. That simplifies this a bit.

Reply:
Resolved in aade935bf6e28eb47483f83d95143e6d3dd8286d